### DISCUSSION TITLE
Welcome to Documentation Discussions!

### DISCUSSION THREAD

**Discussion by corsacca** [DiscipleTools/Documentation#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mikeallbutt** [comment]
I like to use this DT software and have found contributed thoughts and ideas that have helped to improve it. And so that others are use DT, I take the time to write some of the documentation too.
